administrator/components/com_admin/sql/updates/mysql/5.1.0-2023-11-28.sql

```diff
@@ -0,0 +1,2 @@
+INSERT INTO `#__extensions` (`package_id`, `name`, `type`, `element`, `folder`, `client_id`, `enabled`, `access`, `protected`, `locked`, `manifest_cache`, `params`, `custom_data`, `ordering`, `state`) VALUES
+(0, 'plg_schemaorg_article', 'plugin', 'article', 'schemaorg', 0, 1, 1, 0, 0, '', '{}', '', 8, 0);
```

Comment:
```suggestion
(0, 'plg_schemaorg_article', 'plugin', 'article', 'schemaorg', 0, 1, 1, 0, 0, '', '{}', '', 0, 0);
```
In update SQL scripts we normally add new plugins with order number zero so they are shown at the top after the update. But there might be the one or other case where we have done it wrong in past, so not every old update SQL script may be a good example.